src/components/Resources/Schema/SchemasList.tsx

```diff
@@ -112,35 +114,46 @@ const SchemaList = (props: { schemaSelectionCallback: (schemaId: string, schemaD
 		}
 		props.schemaSelectionCallback(schemaId, schemaDetails)
 	}
-	const options = ["All schemas"]
+	// const options = ["All", "Approved", "Requested", "Rejected"]
 
-	const handleFilter = (e: React.ChangeEvent<HTMLSelectElement>) => {
-		if (e.target.value === 'All schemas') {
-			setAllSchemaFlag(true)
-		}
-		else {
-			setAllSchemaFlag(false)
-			getSchemaList(schemaListAPIParameter, false)
-		}
-	};
+	// const handleFilter = (e: React.ChangeEvent<HTMLSelectElement>) => {
+	// 	console.log("Handle filter", e.target.value)
+	// 	if (e.target.value === 'All schemas') {
+	// 		setAllSchemaFlag(true)
```

Comment:
Is this code needed for other development

Reply:
Yes, as we discussed, the Schema list does not require any filter for now as we are showing all the schemas created under ecosystems. So, for now, we can keep this code as it is. I'll remove this code in my next PR after discussion.